### PR TITLE
KATI-145_ForgotPasswordUI

### PR DESCRIPTION
**Pull request by toannguyen71195**

### CODE REVIEW

File: app/src/main/java/com/kms/cura/view/fragment/ForgotPasswordStepThreeFragment.java [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+    }
+
+    private boolean setErrorMessageRePwd() {
+        if (edtRePwd.getText().toString().equals("")) {
+            checkIfRequiredFieldIsEmpty(edittedRePwd, edtRePwd);
+            edittedRePwd = false;
+            return false;
+        }
+        edittedRePwd = true;
+        if (edittedRePwd) {
```
Comment (hieumai) [inline app/src/main/java/com/kms/cura/view/fragment/ForgotPasswordStepThreeFragment.java]: edittedRePwd will always be true here

File: app/src/main/java/com/kms/cura/view/fragment/ForgotPasswordStepThreeFragment.java [hunk 2]
```diff
@@ -0,0 +1,114 @@
...
+    }
+
+    private boolean setErrorMessagePwd() {
+        if (edtPwd.getText().toString().equals("")) {
+            checkIfRequiredFieldIsEmpty(edittedPwd, edtPwd);
+            edittedPwd = false;
+            return false;
+        }
+        edittedPwd = true;
+        if (!checkPwd() && edittedPwd) {
```
Comment (hieumai) [inline app/src/main/java/com/kms/cura/view/fragment/ForgotPasswordStepThreeFragment.java]: edittedPwd will always be true here

File: app/src/main/java/com/kms/cura/view/fragment/ForgotPassworkStepOneFragment.java [hunk 3]
```diff
@@ -0,0 +1,87 @@
...
+    }
+
+    private boolean setErrorMessageEmail() {
+        if (edtEmail.getText().toString().equals("")) {
+            checkIfRequiredFieldIsEmpty(edittedEmail, edtEmail);
+            edittedEmail = false;
+            return false;
+        }
+        edittedEmail = true;
+        if (!checkEmail() && edittedEmail) {
```
Comment (hieumai) [inline app/src/main/java/com/kms/cura/view/fragment/ForgotPassworkStepOneFragment.java]: edittedEmail will always be true here

File: app/src/main/res/layout/fragment_forgot_password_step_three.xml [hunk 4]
```diff
@@ -0,0 +1,69 @@
...
+    <LinearLayout
+        android:layout_width="match_parent"
+        android:layout_height="match_parent"
+        android:orientation="vertical"
+        android:padding="@dimen/padding_16"
+        android:layout_marginTop="@dimen/margin_20">
+        <TextView
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:text="Step 3"
```
Comment (hieumai) [inline app/src/main/res/layout/fragment_forgot_password_step_three.xml]: Put "Step 3" in resource file

File: app/src/main/res/layout/fragment_forgot_password_step_two.xml [hunk 5]
```diff
@@ -0,0 +1,78 @@
...
+            android:layout_height="wrap_content"
+            android:text="@string/security_code"
+            android:textColor="@color/black"
+            android:textSize="@dimen/docText_20"
+            android:padding="@dimen/padding_5"/>
+
+        <TextView
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:text="A code has been sent to your email. Please enter it below:"
```
Comment (hieumai) [inline app/src/main/res/layout/fragment_forgot_password_step_two.xml]: Put "A code has been sent to your email. Please enter it below:" in resource file

**Comment by hieumai** [comment]
OK from me. @huuduy216  and @vtnlinh95  please start verifying this

**Comment by huuduy216** [comment]
@toannguyen71195 
- App stop working when press back button (top left) at step 3
- Pressing the phone's back button bring the view back to log in screen instead of previous step
- The view's name should be "forget password" or "forgot password"

**Comment by hieumai** [comment]
OK from me. @toannguyen71195  please resolve conflicts and rebase so I can merge

**Pull request merged**